icu4c/source/data/BUILDRULES.py

```diff
@@ -176,6 +168,7 @@ def generate_cnvalias(config, glob, common_vars):
     return [
         SingleExecutionRequest(
             name = "cnvalias",
+            category = "cnvalias",
```

Comment:
I can see the arguments both ways.  I think I like having separate categories because:

1. The file extensions are different.
2. There are a lot of things that depend on cnvalias, not just conversion, so you may want to use an exclusion filter on all of conversion but still have cnvalias.